core/src/main/scala/cats/data/StateT.scala

```diff
@@ -1,75 +1,93 @@
 package cats
 package data
 
+import cats.functor.{ Contravariant, Bifunctor, Profunctor }
 import cats.syntax.either._
 
 /**
- * `StateT[F, S, A]` is similar to `Kleisli[F, S, A]` in that it takes an `S`
- * argument and produces an `A` value wrapped in `F`. However, it also produces
- * an `S` value representing the updated state (which is wrapped in the `F`
- * context along with the `A` value.
+ *
+ * `IndexedStateT[F, SA, SB, A]` is a stateful computation in a context `F` yielding
```

Comment:
can you comment about the `Indexed` name here? I don't know the context, though I can understand the rest of the comments.

Reply:
Thanks @johnynek. I refined the scaladoc here. See if that makes it more descriptive.